### DISCUSSION TITLE
Restricted Stock Units (RSU)

### DISCUSSION THREAD

**Discussion by chrishas35** [ghostfolio/ghostfolio#2280] [General]
Is anybody tracking these in ghostfolio?  Trying to decide if it's worthwhile or not, and if so, how. They show up in Fidelity and other aggregator views, so that's really the only reason I'm thinking about wanting to replicate it.  They are essentially worthless to me today, but at a future date when vesting they have value.  That future value would be calculated by the shares vesting and the current stock price.  Not really necessary for net worth views, but perhaps forward looking projections.

**Comment by sfuerte** [comment]
+1 for RSU & ESPP support. While they maybe not so useful in the _current_ net worth, but having an ability to keep track of those would be super helpful.

**Comment by cardom** [comment]
I'm interested as well.
Currently I track them as a normal buy activity when the RSUs has been vested, but with this method the metrics of the investment amount is not accurate, because I didn't spend money to actually buy the stocks, it is an income.
I'm not super happy with this method, if will be possible to track those activities with a different type, like "Income", that does not impact the invest amount, would be really good!

**Comment by jcgonzalezdelgado** [comment]
Any updates here with the non-vested RSUs? I would love to have a way to track them in Ghostfolio :)
